src/runtime/contrib/dnnl/dnnl_json_runtime.cc

```diff
@@ -90,44 +91,95 @@ class DNNLJSONRuntime : public JSONRuntimeBase {
 
  private:
   // Build up the engine based on the input graph.
+  std::map<std::string, tag> layout_dict{
+      {"NCW", tag::ncw},       {"OIW", tag::oiw},     {"GOIW", tag::goiw},   {"NCHW", tag::nchw},
+      {"OIHW", tag::oihw},     {"GOIHW", tag::goihw}, {"NCDHW", tag::ncdhw}, {"OIDHW", tag::oidhw},
+      {"GOIDHW", tag::goidhw}, {"IOHW", tag::iohw},   {"GIOHW", tag::giohw}, {"IODHW", tag::iodhw},
+      {"GIODHW", tag::giodhw},
+  };
+
+  std::map<std::string, dnnl::algorithm> elt_name2algo{
+      {"abs", dnnl::algorithm::eltwise_abs},
+      {"exp", dnnl::algorithm::eltwise_exp},
+      {"log", dnnl::algorithm::eltwise_log},
+      {"sqrt", dnnl::algorithm::eltwise_sqrt},
+      {"round", dnnl::algorithm::eltwise_round},
+      {"logsumexp", dnnl::algorithm::eltwise_logsigmoid},
```

Comment:
Is this mapping exactly the same?

Reply:
Not exactly. `logsumexp` is supposed to be mapped to `logsigmoid`.